noir-projects/aztec-nr/aztec/src/oracle/logs.nr

```diff
@@ -21,7 +21,7 @@ pub unconstrained fn store_private_event_log(
     contract_address: AztecAddress,
     recipient: AztecAddress,
     event_selector: EventSelector,
-    log_content: BoundedVec<Field, MAX_LOG_CONTENT_LEN>,
+    log_content: BoundedVec<Field, MAX_MESSAGE_CONTENT_LEN>,
```

Comment:
log_content should be renamed here to msg_content and the oracle should be renamed as well. I guess we can rename it just to store_private_event.

Reply:
I did not rename the oracle because it _does_ deal with logs (e.g. there's a log index in tx, we search for the log in the node, etc.), but yes this is msg content and not log content.